README.md

```diff
@@ -35,7 +35,7 @@ end
 ### Generic Collection
 
 ```elixir
-<%= svg_image("home") %>
+<%= PhoenixInlineSvg.Helpers.svg_image(@conn, "home") %>
```

Comment:
If you use:

```elixir
  quote do
    ...
    use PhoenixInlineSvg.Helpers, otp_app: :my_app_name
    ...
  end
```

I don't think you need to include `PhoenixInlineSvg.Helpers.`

Reply:
@nikkomiu actually it will not work since you not importing `PhoenixInlineSvg.Helpers` in `__using__` macro. Make a look how phoenix_html doing this: https://github.com/phoenixframework/phoenix_html/blob/master/lib/phoenix_html.ex#L50

Right now __using__ macro only caching images.

Reply:
You are correct, I thought the quote for the import was in there.